frontend/src/components/graphs/StatusCount.js

```diff
@@ -1,44 +1,20 @@
-﻿import React, { useEffect } from 'react';
+﻿import React from 'react';
 import Chart from 'react-apexcharts';
-import { useParams } from 'react-router';
 import { pluck } from 'ramda';
-import { useStateValue } from '../../contexts/state';
+import Loading from '../../components/Loading';
 import { colorTypes } from '../../utils/colorTypes';
 
-const StatusCount = ({ labels }) => {
-    const { seriesId, buildId } = useParams();
-
-    const [{ statusCount }, dispatch] = useStateValue();
-
-    useEffect(() => {
-        const url = `/data/series/${seriesId}/status_counts/?start_from=${buildId}&builds=1`;
-
-        const fetchData = async () => {
-            dispatch({ type: 'setLoadingState', loadingState: true });
-            try {
-                const res = await fetch(url);
-                const json = await res.json();
-                dispatch({ type: 'setLoadingState', loadingState: false });
-                const statusCount = json.status_counts;
-                dispatch({ type: 'setStatusCount', statusCount });
-            } catch (error) {
-                dispatch({ type: 'setErrorState', errorState: error });
-            }
-        };
-        fetchData();
-    }, [buildId, dispatch, seriesId]);
-
+const StatusCount = ({ labels, statusCount }) => {
     const data =
-        statusCount && labels.map(label => pluck(label, statusCount)).flat();
+        statusCount && labels.flatMap(label => pluck(label, statusCount));
 
     const series = data;
```

Comment:
This could just replace the data above right? 🤔

Reply:
yes, i could fix this